src/main.rs

```diff
@@ -3,17 +3,27 @@ mod game;
 
 use std::sync::Mutex;
 
-use crate::game::RoomRegistry;
-
 use actix_web::{body::BoxBody, web, App, HttpResponse, HttpServer};
 use anyhow::Result as AnyhowResult;
+use tokio::sync::mpsc::Sender;
 use tracing_actix_web::TracingLogger;
 
-async fn create_room(state: web::Data<SharedAppState>) -> HttpResponse {
+use crate::game::{RoomDeletionHandler, RoomId, RoomRegistry};
+
+async fn get_rooms(registry: web::Data<Mutex<RoomRegistry>>) -> web::Json<Vec<String>> {
+    let mut room_registry = registry.lock().unwrap();
+    let room_ids = room_registry.list_active_rooms();
+    web::Json(room_ids)
+}
```

Comment:
Probably not going to want to hang onto this handler long term but it is a useful debugging endpoint so I'll keep it around for now